src/index.js

```diff
@@ -154,19 +155,34 @@ function renderPdf(tag) {
   }
 
   const result = {};
+  const isTopLevel = true;
 
   children.forEach(child => {
     const resolvedChild = resolve(child, context);
     result[resolvedChild.elementName] = resolveChildren(
       resolvedChild,
       context,
-      true,
+      isTopLevel,
     );
   });
 
+  const documentFnAttrs = ['footer', 'header'].reduce((memo, key) => {
+    /* eslint-disable-next-line no-param-reassign */
```

Comment:
I think we can skip the assignment of memo[key] here - since we're already spreading `attributes` onto the return value, leaving `documentFnAttrs` to contain only the resolved function header and/or footer.

I would normally not worry about this, but these eslint disable comment are a little distracting, and I'd like to clean them up if we can.